proto/process/agent.proto

```diff
@@ -465,3 +465,35 @@ message Host {
 message DNSEntry {
   repeated string names = 1;
 }
+
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto
+message Pod {
+	string name = 1;
+	string namespace = 2;
+	int64 age = 3;
+	string IP = 4;
+	string nominatedNodeName = 5;
+	string nodeName = 6;
+	int32 restartCount = 7;
```

Comment:
nit, does this need to be an int32?  is there a smaller type that we could use?

Reply:
I matched the original definition https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L902 I guess it could be smaller